src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/SkipLocalsInitAttribute.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    [AttributeUsage(AttributeTargets.Module | AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Constructor | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Event, Inherited = false)]
+    public sealed class SkipLocalsInitAttribute : Attribute
+    {
+        public SkipLocalsInitAttribute()
+        {
+        }
+    }
```

Comment:
We should also switch dotnet/runtime to using this once we have an updated compiler that supports it.  Do you know what the status of that is?
cc: @ViktorHofer

Reply:
Probably be a few weeks before a new compiler with this support is available in dotnet/runtime.

Reply:
Ok, thanks.